pypeit/scripts/view_fits.py

```diff
@@ -51,12 +51,14 @@ def main(args):
 
     # RAW_LRIS??
```

Comment:
Can we structure this routine more generally to just require the spectrograph and then read in the image with the spectrograph class. I don't see why you need to specify raw_lris or raw_deimos anymore given how you restructured things. 

Even better would be if you figured out the instrument from the header thus not requiring the instrument argument.

Reply:
spectrograph now required